users/management/commands/retire_users.py

```diff
@@ -106,6 +107,25 @@ def handle(self, *args, **kwargs):
                 )
                 continue
 
+            resp = bulk_retire_edx_users(user.username)
+            if user.username not in resp["successful_user_retirements"]:
+                self.stderr.write(
+                    self.style.ERROR(
+                        "Could not initiate retirement request on edX for user {user}".format(
+                            user=user
+                        )
+                    )
+                )
+                sys.exit(1)
```

Comment:
Use case: Multiple Users

This would halt the process if edX fails to initiate the retirement process for a single user. The existing implementation continues the process if retirement fails for a single user in MITxOnline. We should continue the process in case of failure from edX as well.